### ISSUE TITLE
Wrong behavior compilator, may be...

### ISSUE THREAD

**Issue by icysnake** [FreeCodeCamp/FreeCodeCamp#11719]
#### Challenge Name

Target the Children of an Element Using jQuery
https://www.freecodecamp.com/challenges/target-the-children-of-an-element-using-jquery
and next challenge.
#### Issue Description

Look at duplicating "#target5" on left-well when refresh page or run tests (ctrl+enter). But when typing code in middle area - everything is good.

#### Browser Information

* Browser Name, Version: Google Chrome 54.0.2840.99
* Operating System: Windows 7
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
 
<script>
  $(document).ready(function() {
    $("#target1").css("color", "red");
    $("#target1").prop("disabled", true);
    $("#target4").remove();
    $("#target2").appendTo("#right-well");
    $("#target5").clone().appendTo("#left-well");
    $("#target1").parent().css("background-color", "red");

  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

 
 
```
#### Screenshot

![s](https://cloud.githubusercontent.com/assets/8858451/20458136/e9407978-aeac-11e6-9db5-030f1c734504.PNG)

**Comment by yaditya** [comment]
I think it's the intended behaviour. This script `$("#target5").clone().appendTo("#left-well");` does it

**Comment by Bouncey** [comment]
Hi @icysnake. Thanks for reporting this, but it is already a known issue (#5072) and has been resolved in the react version of freeCodeCamp (#7430) but it has not gone live yet.

Happy coding!
